runtime/common/RuntimeMLIRCommonImpl.h

```diff
@@ -227,8 +227,10 @@ mlir::LogicalResult verifyOutputRecordingFunctions(llvm::Module *llvmModule) {
         auto callInst = llvm::dyn_cast_or_null<llvm::CallBase>(&inst);
         auto func = callInst ? callInst->getCalledFunction() : nullptr;
         // All call arguments must be constants
-        if (func && failed(verifyConstArguments(callInst)))
-          return mlir::failure();
+        // FIXME: remove the below block once we can confirm that const argument
+        // is not a requirement. if (func &&
+        // failed(verifyConstArguments(callInst)))
+        //   return mlir::failure();
```

Comment:
The formatting did something weird here, but regardless, I'd recommend limiting keeping this check for the base profile.
```suggestion
        if (isBaseProfile && func && failed(verifyConstArguments(callInst)))
          return mlir::failure();
```